tests/functional/clients/standalone/test_package_list.py

```diff
@@ -1,5 +1,5 @@
 # from ['image']['imagedata']['analysis_report']['package_list']
-from .fixtures import gems
+from .fixtures import gems, pypkgs
```

Comment:
Should we start to create separate test_package modules per language as this file is going to become very large and if there are additional fields to test it will just add to the size?

Reply:
@Toure these are already separate files:

```
$ tree standalone/fixtures/
standalone/fixtures/
├── __init__.py
├── gems.py
├── npms.py
└── pypkgs.py

0 directories, 4 files
```

So the sizes are already small. Do you mean I need to do something else with these?

Reply:
sorry, the comment should have been against the test_package_list.py file itself to be broken out into language-specific modules.

Reply:
Oh ok, I see what you mean. Would it be OK to get those non-functional changes (along with @wagoodman 's suggestions) in a separate PR?